tests/tck/features/go/GoYieldVertexEdge.feature

```diff
@@ -0,0 +1,1753 @@
+      | [:serve "Tony Parker"->"Spurs" @0 {end_year: 2018, start_year: 1999}]               |
+      | [:serve "LaMarcus Aldridge"->"Spurs" @0 {end_year: 2019, start_year: 2015}]         |
+      | [:serve "LaMarcus Aldridge"->"Trail Blazers" @0 {end_year: 2015, start_year: 2006}] |
+      | [:serve "Tim Duncan"->"Spurs" @0 {end_year: 2016, start_year: 1997}]                |
+      | [:serve "Manu Ginobili"->"Spurs" @0 {end_year: 2018, start_year: 2002}]             |
+    When executing query:
+      """
+      GO FROM "No Exist Vertex Id" OVER like YIELD dst(edge) as id |
+      GO FROM $-.id OVER like YIELD dst(edge) as id |
+      GO FROM $-.id OVER serve YIELD $$ as dst
+      """
+    Then the result should be, in any order, with relax comparison:
+      | dst |
+
+  Scenario: In expression
+    When executing query:
+      """
+      GO FROM 'Tony Parker' OVER like WHERE like._dst IN ['Tim Duncan', 'Danny Green'] YIELD $$ as dst, tags($$), tags($^)
+      """
+    Then the result should be, in any order, with relax comparison:
+      | dst                                                                                                         | tags($$)               | tags($^)   |
+      | ("Tim Duncan" :bachelor{name: "Tim Duncan", speciality: "psychology"} :player{age: 42, name: "Tim Duncan"}) | ["bachelor", "player"] | ["player"] |
+    When executing query:
+      """
+      GO FROM 'Tony Parker' OVER like WHERE like._dst IN ['Tim Duncan', 'Danny Green'] YIELD $^ as src
+      """
+    Then the result should be, in any order, with relax comparison:
+      | src                                                   |
+      | ("Tony Parker" :player{age: 36, name: "Tony Parker"}) |
+
+  Scenario: assignment simple
+    When executing query:
+      """
+      $var = GO FROM "Tracy McGrady" OVER like YIELD dst(edge) as id;
+      GO FROM $var.id OVER like YIELD $$ as dst
+      """
+    Then the result should be, in any order, with relax comparison:
+      | dst                                                               |
+      | ("Tracy McGrady" :player{age: 39, name: "Tracy McGrady"})         |
+      | ("LaMarcus Aldridge" :player{age: 33, name: "LaMarcus Aldridge"}) |
+
+  Scenario: assignment pipe
+    When executing query:
+      """
+      $var = (GO FROM "Tracy McGrady" OVER like YIELD dst(edge) as id | GO FROM $-.id OVER like YIELD dst(edge) as id);
+      GO FROM $var.id OVER like YIELD dst(edge) as dst
+      """
+    Then the result should be, in any order, with relax comparison:
+      | dst           |
+      | "Kobe Bryant" |
+      | "Grant Hill"  |
+      | "Rudy Gay"    |
+      | "Tony Parker" |
+      | "Tim Duncan"  |
+
+  Scenario: pipe only yield input prop
+    When executing query:
+      """
+      GO FROM "Tracy McGrady" OVER like YIELD dst(edge) as vid |
+      GO FROM $-.vid OVER like YIELD src(edge) as id
+      """
+    Then the result should be, in any order, with relax comparison:
```

Comment:
```
GO FROM "Tim" OVER like YIELD $^ AS src, id($$) AS dst | 
GO FROM $-.dst OVER like YIELD $$ AS dest, $-.src
```

Reply:
Maybe we could support the following case:

```
GO FROM "Tim" OVER like YIELD $^ AS src, $$ AS dst | 
GO FROM id($-.dst) OVER like YIELD $$ AS dest, $-.src
```

Reply:
add